examples/docker-agent/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6
+
+RUN pip install --trusted-host pypi.python.org Flask~=0.12
+
+ADD ./examples/docker-agent /agent
+
+# @TODO to be replaced with `pip install pommerman`
+ADD . /pommerman
+RUN cd /pommerman && pip install .
+# end @TODO
+
+EXPOSE 10080
+
```

Comment:
Did you test that this Dockerfile works with run_battle?

Reply:
It was definitely working earlier. Somehow after the new conflict resolution with `master`, it seems to be crashing.